packages/eth/src/websocket/connection.rs

```diff
@@ -135,6 +155,53 @@ impl WsConnection {
     async fn _balance(&self, address: H160) -> Result<U256> {
         Ok(self.provider.get_balance(address, None).await?)
     }
+
+    async fn prepare_blob_tx(&self, blob_versioned_hashes: Vec<H256>) -> Result<BlobTransaction> {
+        let address = self.blob_pool_wallet.address();
+        let nonce = self.provider.get_transaction_count(address, None).await?;
+
+        let (max_fee_per_gas, max_priority_fee_per_gas) =
+            self.provider.estimate_eip1559_fees(None).await?;
+
+        // Gas limit should be 21000, otherwise we'll have to estimate it
```

Comment:
A standard ETH transfer requires a gas limit of 21000. Unless we have to use calldata (depending on how fraud proving will look like), the blob transaction should always correspond to that limit since the blob data is counted separately.
We can potentially save the network call here

Reply:
Ah, I see what the strategy here is. That's a good trade-off, IMO.